Demo/Demo/Views/FeatureBaseViewControllerRepresentable.swift

```diff
@@ -0,0 +1,15 @@
+import SwiftUI
+
+@available(iOS 13, *)
+struct FeatureBaseViewControllerRepresentable: UIViewControllerRepresentable {
+
+    var baseViewModel: BaseViewModel
+
+    func makeUIViewController(context: Context) -> UIViewController {
```

Comment:
I don't have much knowledge about SwiftUI so this maybe a silly question :smile: 
But I don't see we use this anywhere, what is this for? or is this something required by SwiftUI?

Reply:
Great question - so `FeatureBaseViewControllerRepresentable` is used by `SwiftUICardDemo` when we add the `FeatureBaseViewController` to our SwiftUI view here:
```
FeatureBaseViewControllerRepresentable(baseViewModel: baseViewModel)
```
Under the hood, SwiftUI calls the `makeUIViewController` function when creating this view as part of the `UIViewControllerRepresentable` protocol. Essentially this is our abstraction layer for our UIViewController to allow us to represent this view within SwiftUI. If you add a breakpoint to this function then spin up the demo and select `SwiftUI` you will see the system calls this function automatically when needed. 😄